docs/quickstart.md

```diff
@@ -0,0 +1,164 @@
+<!--[metadata]>
++++
+title = "Quickstart Guide"
+description = "Getting started with Docker Compose"
+keywords = ["documentation, docs,  docker, compose, orchestration, containers"]
+[menu.main]
+parent="smn_workw_compose"
+weight=3
++++
+<![end-metadata]-->
+
+
+## Quick start
+
+Let's get started with a walkthrough of getting a simple Python web app running
+on Compose. It assumes a little knowledge of Python, but the concepts
+demonstrated here should be understandable even if you're not familiar with
+Python.
+
+### Installation and set-up
+
+First, [install Docker and Compose](install.md).
+
+Next, you'll want to make a directory for the project:
+
+    $ mkdir composetest
+    $ cd composetest
+
+Inside this directory, create `app.py`, a simple Python web app that uses the Flask
+framework and increments a value in Redis. Don't worry if you don't have Redis installed, docker is going to take care of that for you when we [define services](#define-services):
```

Comment:
"docker is going to take care of that" is a bit awkward; also avoid future tense and "we"